Update default template with UG links and useful information (#2225)
MarkBind/markbind: packages/core/template/default/index.md

<frontmatter>
  title: Home Page
  layout: default.md
  pageNav: 4

>   pageNavTitle: "Chapters of This Page"

Reviewer: perhaps a more appropriate nav title here

Author: Fixed in [945ea73](https://github.com/MarkBind/markbind/pull/2225/commits/945ea738078da9000f4299903da634c4db9dee5e)!

>   pageNavTitle: "Placeholder Topic Pages"

Reviewer: ```suggestion
  pageNavTitle: "Topics"
```

Author: [fab85db](https://github.com/MarkBind/markbind/pull/2225/commits/fab85db23ed5ba23eacde0a78dc99c45d63d69c7)

</frontmatter>

<br>

<div class="bg-primary text-white px-2 py-5 mb-4">
  <div class="container">
    <h1 class="display-5 no-index">Great!<br>You've just initialized a MarkBind site.</h1>
    <p class="lead">Let's get started...</p>
  </div>
</div>

---

> ## What did I just do?

Reviewer: ```suggestion
## What did you just do?
```

Author: Fixed in [945ea73](https://github.com/MarkBind/markbind/pull/2225/commits/945ea738078da9000f4299903da634c4db9dee5e)!

> You have just initialized a _default_ MarkBind site, which is equipped with a set of core features of a MarkBind site like Site and Page Navigation, as well as some links to our User Guide, to help you get started more conveniently!

Reviewer: ```suggestion
You have just initialized a _default_ MarkBind site! It is equipped with a set of core features, including site and page navigation. Additionally, we have included some handy links to our User Guide to help you get started quickly and easily.
```

Author: Thanks for the suggestion! Updated in [fab85db](https://github.com/MarkBind/markbind/pull/2225/commits/fab85db23ed5ba23eacde0a78dc99c45d63d69c7).

<box type="tip">

> If you were intending to convert an existing GitHub wiki or a docs folder into MarkBind, use the "--convert" flag instead. See <a href="https://markbind.org/userGuide/markBindInTheProjectWorkflow.html#converting-existing-project-documentation-wiki" target="_blank">User Guide: MarkBind in the Project Workflow</a> for more information.

Reviewer 2: ```suggestion
If you were intending to convert an existing GitHub wiki or a docs folder into MarkBind, use the `--convert` flag instead. See <a href="https://markbind.org/userGuide/markBindInTheProjectWorkflow.html#converting-existing-project-documentation-wiki" target="_blank">User Guide: MarkBind in the Project Workflow</a> for more information.
```
Might be better to use the inline code format? Same for the --template below

Author: Thanks for the suggestion! Updated in [eface18](https://github.com/MarkBind/markbind/pull/2225/commits/eface184bfc22fc7a9db7f55e82882f47f018d5e).

> If you want to start with a <tooltip content="i.e. without any content">_minimal_</tooltip> template instead, use the "--template" flag to initialize a minimal site instead of the default. See <a href="https://markbind.org/userGuide/templates.html" target="_blank">User Guide: Templates</a> for more information.

Reviewer 3: ```suggestion
If you want to start with a <tooltip content="i.e. without any content">_minimal_</tooltip> template instead, use the "--template" flag with "minimal" option to initialize a minimal site instead of the default. See <a href="https://markbind.org/userGuide/templates.html" target="_blank">User Guide: Templates</a> for more information.
```

Author: Thanks for the suggestion! Updated in [fab85db](https://github.com/MarkBind/markbind/pull/2225/commits/fab85db23ed5ba23eacde0a78dc99c45d63d69c7).

</box>

---

## Navigating this site

> This _default_ site comes pre-configured with the core <a href="https://markbind.org/userGuide/components/navigation.html#navigation-components" target="_blank">Navigation components</a>: a <tooltip content="Site Navigation">**siteNav**</tooltip>, a <tooltip content="Page Navigation">**pageNav**</tooltip>, a <tooltip content="Navigation Bar">**NavBar**</tooltip>, and a **Search Bar**</tooltip>. To help you get started with the **siteNav**, we have included <tooltip content="Topic 1, Topic 2, Topic 3, Topic 3a, Topic 3b">five dummy placeholder pages</tooltip>. The **NavBar** also comes with a placeholder slot for your custom Logo.

Reviewer 3: ```suggestion
This _default_ site comes pre-configured with the core <a href="https://markbind.org/userGuide/components/navigation.html#navigation-components" target="_blank">Navigation components</a>: a <tooltip content="Site Navigation">**siteNav**</tooltip>, a <tooltip content="Page Navigation">**pageNav**</tooltip>, a <tooltip content="Navigation Bar">**NavBar**</tooltip>, and a **Search Bar**. To help you get started with the **siteNav**, we have included <tooltip content="Topic 1, Topic 2, Topic 3, Topic 3a, Topic 3b">five dummy placeholder pages</tooltip>. The **NavBar** also comes with a placeholder slot for your custom Logo.
```

Author: Thanks for the find! I didn't notice that :sweat_smile: 

Updated in [fab85db](https://github.com/MarkBind/markbind/pull/2225/commits/fab85db23ed5ba23eacde0a78dc99c45d63d69c7).

---

## Guide to MarkBind

> To see the capability of MarkBind in action, feel free to take a look at some of the websites built using MarkBind on our <a href="https://markbind.org/userGuide/gettingStarted.html" target="_blank">Showcase</a> page. 

Reviewer: wrong link

For more information on how to work with MarkBind sites and to add content, refer to our comprehensive <a href="https://markbind.org/userGuide/gettingStarted.html" target="_blank">User Guide</a>. 

<box type="info">

If you are interested in contributing to MarkBind, you can refer to our <a href="https://markbind.org/devdocs/devGuide/devGuide.html" target="_blank">Developer Guide</a> as well!

</box>

> <panel header="**Good starting points in our User Guide:**" expanded no-close>

Reviewer 2: ```suggestion
<panel header="**Good starting points in our User Guide**" expanded no-close>
```

Author: Thanks for the suggestion! Updated in [eface18](https://github.com/MarkBind/markbind/pull/2225/commits/eface184bfc22fc7a9db7f55e82882f47f018d5e).

##### **User Guide: Authoring Contents**

> Learn about the variety of syntax schemes, formats, and custom MarkBind components that you can use in your MarkBind site.

> More info in: _<a href="https://markbind.org/userGuide/authoringContents.html" target="_blank">User Guide → Authoring Contents</a>_

Reviewer: This link and line 65 is the same, is this a mistake?

Author: Fixed in [945ea73](https://github.com/MarkBind/markbind/pull/2225/commits/945ea738078da9000f4299903da634c4db9dee5e)!

---

##### **User Guide: Working with Sites**

> > Learn how to modify site properties, apply themes, and enable / disable plugins for your MarkBind site.

Author: Thanks for the suggestion! Updated in [fab85db](https://github.com/MarkBind/markbind/pull/2225/commits/fab85db23ed5ba23eacde0a78dc99c45d63d69c7).

More info in: _<a href="https://markbind.org/userGuide/workingWithSites.html" target="_blank">User Guide → Working with Sites</a>_

---

##### **User Guide: Full Syntax Reference**

> Refer to our Full Syntax Reference page to find a specific feature or component that you want to use in your MarkBind site.

More info in: _<a href="https://markbind.org/userGuide/fullSyntaxReference.html" target="_blank">User Guide → Full Syntax Reference</a>_

> </panel>

Reviewer 2: Can we add some line breaks after the panel here so that the panel isn't tight on the bottom of the page?
<img width="751" alt="Screenshot 2023-03-29 at 4 51 45 PM" src="https://user-images.githubusercontent.com/61113575/228486202-6cce6cf0-77e1-4a43-8269-45d3e625bdec.png">

Author: Thanks for the suggestion! Updated in [eface18](https://github.com/MarkBind/markbind/pull/2225/commits/eface184bfc22fc7a9db7f55e82882f47f018d5e).

<div class="bg-primary text-white py-5 mb-4">
  <div class="container">

>     <h1 class="display-4 no-index">Great!<br>You just initialized a MarkBind site.</h1>

Reviewer 2: Minor nit!
```suggestion
    <h1 class="display-4 no-index">Great!<br>You've just initialized a MarkBind site.</h1>
```

    <p class="lead">Let's get started...</p>
  </div>
</div>

<hr>

> <h5><strong>User Guide: Authoring Contents</strong></h5>

Reviewer 4: Instead of using pure html in the whole file, can we use a mix of html and markdown instead? This might be easier for first time users and is also one of our selling points. 

I think the headings should at least be done in markdown instead of html

Author: That's a good idea! I've changed the headings to markdown in [d5317d0](https://github.com/MarkBind/markbind/pull/2225/commits/d5317d095a7a908d8a582e36e54a7751e4889a53).

<blockquote>

Learn about the variety of syntax schemes, formats, and custom MarkBind components that you can use in your MarkBind site.

</blockquote>

<span class="indented">More info in: _<a href="https://markbind.org/userGuide/authoringContents.html" target="_blank">User Guide → Authoring Contents</a>_</span>

<hr>

##### **User Guide: Working with Sites**

> <blockquote>

Reviewer: Any problem with `>` instead of `<blockquote>` ? I would prefer Markdown over direct HTML. Same for other content in this page

Author: Absolutely no problem! I simply copied what was being done in the [Authoring Contents file](https://github.com/MarkBind/markbind/blob/master/docs/userGuide/authoringContents.md). 

Markdown definitely does make it cleaner - in [44f4454](https://github.com/MarkBind/markbind/pull/2225/commits/44f44542396dd4ab489d92fbe067fcb3498f174a), I've changed the `<blockquote>` to `>`, replaced `<hr>` with `---`, and removed the `<span>`. For now, I kept the blue heading container (to keep the style) and the `<a>` tags (to make use of `target="_blank"`).
